modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/PolicySpecifiedDefinition.cs

```diff
@@ -0,0 +1,37 @@
+﻿using JetBrains.Annotations;
+using Volo.Abp;
+using Volo.Abp.Localization;
+
+namespace Volo.CmsKit.Domain.Volo.CmsKit
+{
+    public abstract class PolicySpecifiedDefinition
+    {
+        protected PolicySpecifiedDefinition()
+        {
+        }
+
+        public PolicySpecifiedDefinition(
+            [CanBeNull] ILocalizableString displayName = null,
+            [CanBeNull] string createPolicy = null,
+            [CanBeNull] string updatePolicy = null,
+            [CanBeNull] string deletePolicy = null)
+        {
+            DisplayName = displayName;
+            CreatePolicy = createPolicy;
+            DeletePolicy = deletePolicy;
+            UpdatePolicy = updatePolicy;
+        }
+
+        [CanBeNull]
+        public virtual ILocalizableString DisplayName { get; }
```

Comment:
Why do we need `DisplayName` in this type? I guess people can use this to show special display names for entity types. Maybe we should introduce another base type or interface for this. Can you work on this? I think we can improve the design here.